evergreen.yml

```diff
@@ -672,6 +681,9 @@ functions:
         script: |
           ${prepare_shell}
 
+          # change mongosql version in core/Cargo.toml from branch to tag
+          sed -i "s/branch = \"main\"/tag = \"${MONGOSQL_VERSION}\"/" core/Cargo.toml
```

Comment:
I think we should do this as part of "set and check package version".
First, it will be only in one place instead of 2 which is better for maintainability. 
Second, it might impact the SBOM content. We want to have to correct version reported if it appears as a dependency (it should). 
Unfortunately, I cannot tell because the SBOM generation was disabled in https://github.com/mongodb/mongo-odbc-driver/pull/263. Can you check if the `sbom` task can be re-enabled. I agree that we don't need to produce the SSDLC release artifacts, but it would be good to have the sbom task still running. This will make sure that if we introduce new/different libraries, we are also clear.